src/frontend/components/DashboardVideoLiveControlPane/customs/DashboardVideoLiveConfirmationModal/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Button, Layer, ResponsiveContext, Text } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { RoundCrossSVG } from 'components/SVGIcons/RoundCrossSVG';
+import { theme } from 'utils/theme/theme';
+
+const messages = defineMessages({
+  confirmButtonLabel: {
+    defaultMessage: 'Confirm',
+    description: 'Label of the confirming button',
+    id: 'components.DashboardVideoLiveConfirmationModal.confirmButtonLabel',
+  },
+  cancelButtonLabel: {
+    defaultMessage: 'Cancel',
+    description: 'Label of the cancelling button',
+    id: 'components.DashboardVideoLiveConfirmationModal.cancelButtonLabel',
+  },
+});
+
+const StyledTitleText = styled(Text)`
+  font-family: 'Roboto-Medium';
+`;
+
+const StyledText = styled(Text)`
+  font-family: 'Roboto-Regular';
+  line-height: 20px;
+`;
+
+interface DashboardVideoLiveConfirmationModalProps {
+  text: string;
+  title: string;
+  onModalCloseOrCancel: () => void;
+  onModalConfirm: () => void;
+}
+
+export const DashboardVideoLiveConfirmationModal = ({
+  text,
+  title,
+  onModalCloseOrCancel,
+  onModalConfirm,
+}: DashboardVideoLiveConfirmationModalProps) => {
+  const intl = useIntl();
+  const size = React.useContext(ResponsiveContext);
+
+  return (
+    <Layer
+      onEsc={onModalCloseOrCancel}
+      onClickOutside={onModalCloseOrCancel}
+      responsive={false}
+      style={{
+        width: size === 'small' ? '95%' : '500px',
```

Comment:
That's huge difference. Are you sure about this values ?

Reply:
For info, this component comes from the PR on the shared live media : https://github.com/openfun/marsha/blob/9f991d654dfb754566af8ca8dd71c93ec49becd7/src/frontend/components/DashboardVideoLiveControlPane/customs/DashboardVideoLiveConfirmationModal/index.tsx

I think the values are correct because this is what I wanted : a almost full page modal when the screen is small, a medium fixed sized modal otherwise